easy-coding-standard.yml

```diff
@@ -1,11 +1,177 @@
 imports:
-    - { resource: 'packages/coding-standards/easy-coding-standard.yml' }
     - { resource: 'packages/*/easy-coding-standard.yml' }
     - { resource: 'project-base/easy-coding-standard.yml' }
 
+services:
+    _defaults:
+        autowire: true
+
+    # helper services ↓
+    PhpCsFixer\Tokenizer\Analyzer\FunctionsAnalyzer: ~
+    Shopsys\CodingStandards\Helper\PhpToDocTypeTransformer: ~
+    Shopsys\CodingStandards\Helper\FqnNameResolver: ~
+    PhpCsFixer\Tokenizer\Analyzer\NamespaceUsesAnalyzer: ~
+    Symplify\CodingStandard\TokenRunner\Analyzer\FixerAnalyzer\IndentDetector: ~
+
+    # Slevomat Checkers
+    SlevomatCodingStandard\Sniffs\Commenting\InlineDocCommentDeclarationSniff: ~
+    SlevomatCodingStandard\Sniffs\TypeHints\NullableTypeForNullDefaultValueSniff: ~
+    SlevomatCodingStandard\Sniffs\TypeHints\ReturnTypeHintSpacingSniff: ~
+
+    # Shopsys Checkers
+    Shopsys\CodingStandards\CsFixer\ForbiddenDumpFixer: ~
```

Comment:
Because there are many changes in monorepo vs. shopsys/coding-standard I am not includin it any more and it stays duplicated and udated in monorepo.

Main reason to doing so is to keep backward compatibility.

Reply:
I would prefer to have only different directives in monorepo configuration to keep it more readable and to avoid different rules by mistake

Reply:
Ok, I fixed this one. e77a242d6c84d9fd615063719f2fd7ff57b36b26

Reply:
nice, can't wait to test it :)